### PR TITLE
created first structure of cv-jedi-minimalist and his header section

### PR DESCRIPTION
**Pull request by AbelDeTena**
I have created a small structure of directories:
- cv-jedi-minimalism
 - html-parts
  - html-document-start (including his own styles)
  - html-document-end
  - header-element-start
  - header-section (first approach)
  - header-element-end

### CODE REVIEW

File: packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.ejs [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        --spacing_8: calc(var(--spacing_base) * 2);
+        --spacing_12: calc(var(--spacing_base) * 3);
+        --spacing_16: calc(var(--spacing_base) * 4);
+        --spacing_24: calc(var(--spacing_base) * 6);
+        --spacing_32: calc(var(--spacing_base) * 8);
+        --spacing_48: calc(var(--spacing_base) * 12);
+        --spacing_64: calc(var(--spacing_base) * 16);
+
+        /* colors */
+        --primary_color: #11383a; /* only this color can be changed  */
```
Comment (juanpms2) [inline packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.ejs:28]: Use selected color here, please

File: packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.part.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import ejs from 'ejs';
+import { HexColor } from '@/model';
```
Comment (juanpms2) [inline packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.part.ts:2]: This CV need a new HexColor enum with its values.

File: packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.ejs (hunk 1 above)
Comment (AbelDeTena) [reply packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.ejs]: Replaced by:

   --primary_color:<%= color -%>; /* only this color can be changed  */

File: packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.part.ts (hunk 2 above)
Comment (juanpms2) [reply packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/html-document-start/html-document-start.part.ts]: Yes, but not only this, you need a new mapper and use it. Check if the colors are corect when you select a color know.

**Pull request merged**